redos/7.3/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM centos:7 as prepare
+MAINTAINER Sergei Vorontsov "piligrim@rootnix.net"
+
+ENV REPO_URL=http://mirror.yandex.ru/redos/7.3/x86_64/os/ \
+    CHROOT_DIR="/chroot"
+
+RUN mkdir -p $CHROOT_DIR
+
+#Download any available meta package redos-release
```

Comment:
Please add a space between `#` and `Download`.

Reply:
Done.